fuel-coroutines/README.md

```diff
@@ -3,7 +3,7 @@ The coroutines extension package for [`Fuel`](../README.md).
 
 ## Installation
 
-You can [download](https://bintray.com/kittinunf/maven/Fuel-Android/_latestVersion) and install `fuel-coroutines` with `Maven` and `Gradle`. The coroutines package has the following dependencies:
+You can [download](https://mvnrepository.com/artifact/com.github.kittinunf.fuel/fuel-coroutines/latest) and install `fuel-coroutines` with `Maven` and `Gradle`. The coroutines package has the following dependencies:
```

Comment:
🤔 What do you think about this link, instead? https://bintray.com/kittinunf/maven/Fuel-Android/2.2.1#files/com%2Fgithub%2Fkittinunf%2Ffuel%2Ffuel-coroutines

Reply:
doesn't seem to work any better, that link doesn't go right to the files. Takes you to the same landing page.